Woopra/Woopra-iOS
# Rename framework to AppierWoopra and keep the import module as-is - Woopra

# Context
There is another old CocoaPods framework named `Woopra` since 2015 earlier than `Woopra-iOS` (2017). 
To avoid ambiguous pod install, this PR defines the CocoaPods framework as `AppierWoopra` and keeps the import SDK name as `Woopra`.

# How to verify this
1. Install the local pod by modifying the Podfile in your test project
```
pod 'AppierWoopra', :path => '/path/to/local/pod'
```
2. Execute `pod install`
3. Confirm `import Woopra` works
